ports/doddgoestothemuseum/port.json

```
@@ -0,0 +1,45 @@
+{
+  "version": 4,
+  "name": "doddgoestothemuseum.zip",
+  "items": [
+    "Dodd Goes to the Museum.sh",
+    "doddgoestothemuseum"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Dodd Goes to the Museum",
+    "porter": [
+      "Doronimmo"
+    ],
+    "desc": "Explore Dodd's local museum in this short, casual and relaxing adventure! Visit various exhibits within the museum's four wings and talk to various characters along the way!",
+    "desc_md": null,
+    "inst": "Ready to run! Thanks Schwegg (https://www.tumblr.com/schwender-exe) for allowing us to distribute your amazing game! You can get it on itch.io (https://schwegg.itch.io/dodd-goes-to-the-museum) and steam (https://store.steampowered.com/app/2247050/Dodd_Goes_to_the_Museum/)",
```

```suggestion
    "inst": "Ready to run!",
```
Thank you notes belong in the readme not the instructions.